frontend/apps/reader/modules/readerhighlight.lua

```diff
@@ -466,8 +468,20 @@ function ReaderHighlight:addToMainMenu(menu_items)
             sub_item_table = self:genPanelZoomMenu(),
         }
     end
+
+    -- main menu Search
+    menu_items.search_settings = { -- submenu with Dict, Wiki, Translation settings
+        text = _("Settings"),
+    }
```

Comment:
May be this one should better be in readerdictionary, which is the one that brings into it the most subitems and the first in that search menu? (ReaderHighlight feels a bit unrelated).

Reply:
What about inconsistency `Dictionary lookup history` vs `Wikipedia history`?

Reply:
Dunno when/where this difference comes from :/ @Frenzie will tell us if/how we should fix that.

Reply:
I don't know where it came from, but I think "Dictionary history" might be a bit ambiguous?